README.md

```diff
@@ -1,14 +1,14 @@
 >Please help us improve and share your feedback! If you find better tutorials or links, please share them by opening a Pull Request.
 
-# HackYourJavaScript
+# JavaScript3
```

Comment:
Or maybe `HackYourFuture JavaScript 3` instead?

Reply:
yep I like it.